Disable device provisioning

If device provisioning is enabled, EAS needs admin permissions on your
device. I'm not sure if that is a default setting that we want in a
Debian package.

Index: z-push/src/config.php
===================================================================
--- z-push.orig/src/config.php	2017-01-04 15:07:08.410251144 +0100
+++ z-push/src/config.php	2017-01-04 15:10:28.958961696 +0100
@@ -132,7 +132,7 @@
  *  Mobile settings
  */
     // Device Provisioning
-    define('PROVISIONING', true);
+    define('PROVISIONING', false);
 
     // This option allows the 'loose enforcement' of the provisioning policies for older
     // devices which don't support provisioning (like WM 5 and HTC Android Mail) - dw2412 contribution
